src/openforms/submissions/form_logic.py

```diff
@@ -81,3 +80,27 @@ def evaluate_form_logic(
     step._form_logic_evaluated = True
 
     return configuration
+
+
+def check_submission_logic(submission):
+    logic_rules = FormLogic.objects.filter(
+        form=submission.form,
+        actions__contains=[{"action": {"type": "step-not-applicable"}}],
```

Comment:
can we have the action types defined in a constant somewhere and use those labels instead of magic strings?